src/stobj.c

```diff
@@ -0,0 +1,979 @@
+    func_8006B8E4(decodedStageLzPtr->animGroups, decodedStageLzPtr->animGroupCount);
+    func_8006B9E4(decodedStageLzPtr->animGroups, decodedStageLzPtr->animGroupCount);
+    func_8006C7BC(decodedStageLzPtr->animGroups, decodedStageLzPtr->animGroupCount);
+}
+
+#pragma force_active on
+char *asdfasdf[] =
+{
+    "SOT_BUMPER",
+    "SOT_JAMABAR",
+    "SOT_GOALTAPE",
+    "SOT_GOALBAG",
+    "SOT_GOALBAG_EXMASTER",
+    "SOT_MF_PNL_BUMPER",
+    "SOT_MF_PNL_ELECTRAP",
+    "SOT_MF_BULLET_TEST",
+    "SOT_MF_BOX",
+    "SOT_BUMPER_BGSPECIAL",
+    "SOT_NAMEENT_BTN",
+};
+#pragma force_active reset
+
+void (*stobjInitFuncs[])(struct Stobj_ *) =
+{
+    stobj_bumper_init,
+    stobj_jamabar_init,
+    stobj_goaltape_init,
+    stobj_goalbag_init,
+    stobj_goalbag_exmaster_init,
+    stobj_dummy_init,
+    stobj_dummy_init,
+    stobj_dummy_init,
+    stobj_dummy_init,
+    stobj_bumper_bgspecial_init,
+    stobj_nameent_btn_init,
+    NULL,
+};
+
+void (*stobjMainFuncs[])(struct Stobj_ *) =
+{
+    stobj_bumper_main,
+    stobj_jamabar_main,
+    stobj_goaltape_main,
+    stobj_goalbag_main,
+    stobj_goalbag_exmaster_main,
+    stobj_dummy_main,
+    stobj_dummy_main,
+    stobj_dummy_main,
+    stobj_dummy_main,
+    stobj_bumper_bgspecial_main,
+    stobj_nameent_btn_main,
+    NULL,
+};
+
+void (*stobjDrawFuncs[])(struct Stobj_ *) =
+{
+    stobj_bumper_draw,
+    stobj_jamabar_draw,
+    stobj_goaltape_draw,
+    stobj_goalbag_draw,
+    stobj_goalbag_exmaster_draw,
+    stobj_dummy_draw,
+    stobj_dummy_draw,
+    stobj_dummy_draw,
+    stobj_dummy_draw,
+    stobj_bumper_bgspecial_draw,
+    stobj_nameent_btn_draw,
+    NULL,
+};
+
+void (*stobjCollisionFuncs[])(struct Stobj_ *, struct PhysicsBall *) =
+{
+    stobj_bumper_coli,
+    stobj_jamabar_coli,
+    stobj_goaltape_coli,
+    stobj_goalbag_coli,
+    stobj_goalbag_exmaster_coli,
+    stobj_dummy_coli,
+    stobj_dummy_coli,
+    stobj_dummy_coli,
+    stobj_dummy_coli,
+    stobj_bumper_bgspecial_coli,
+    stobj_nameent_btn_coli,
+    NULL,
+};
+
+void (*lbl_801BE1F0[])(struct Stobj_ *) =
+{
+    func_8006C13C,
+    func_8006C6C8,
+    func_8006DD98,
+    func_8006F3A0,
+    func_8006F428,
+    func_8006C7B4,
+    func_8006C7B4,
+    func_8006C7B4,
+    func_8006C7B4,
+    func_8006C404,
+    func_800AFC14,
+    NULL,
+};
+
+void (*lbl_801BE220[])(struct Stobj_ *) =
+{
+    func_8006C140,
+    func_8006C6CC,
+    func_8006DD9C,
+    func_8006F3A4,
+    func_8006F42C,
+    func_8006C7B8,
+    func_8006C7B8,
+    func_8006C7B8,
+    func_8006C7B8,
+    func_8006C408,
+    func_800AFC18,
+    NULL,
+};
+
+#pragma force_active on
+
+char string_STATUS___s_n_2[] = "STATUS: %s\n";
+char lbl_801BE25C[] =
+{
+    0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x0A,
+    0x00
+};
+char string_TYPE___s_n_2[] = "TYPE: %s\n";
+char string_PART_ID___d_n[] = "PART ID: %d\n";
```

Comment:
Were AnimGroups just called "parts" internally?

Reply:
That's definitely a possibility, though I can't find anything that uses that string. It may have been left over from a function that got stripped by the linker.